plugins/modules/swupd.py

```diff
@@ -30,7 +30,7 @@
     type: str
   format:
     description:
-      - The format suffix for version file downloads. For example [1,2,3,staging,etc]. If not specified, the default format is used.
+      - The format suffix for version file downloads. For example V([1,2,3,staging,...]). If not specified, the default format is used.
```

Comment:
I saw this earlier and first wanted to suggest this as well, but then I wasn't sure what exactly the documentation wants to say here. Are `1`, `2`, `3`, `staging`, etc. different example values for `format`? Or is `[1,2,3,staging,...]` a valid value for `format`?

Reply:
According to https://github.com/clearlinux/swupd-client/blob/master/docs/swupd.1.rst it should be one of `1`, `2`, `3`, etc., or the special value `staging`.

How about:
```suggestion
      - The format suffix for version file downloads. For example V(1), V(2), V(3), and so on, or the special value V(staging). If not specified, the default format is used.
```

Reply:
Sounds perfect, thanks!